draft-sharabayko-srt.md

```diff
@@ -1531,113 +1544,123 @@ The INDUCTION phase serves only to set a cookie on the Listener so that it
+connection MUST be rejected with rejection reason `SRT_REJ_ROGUE`. This is, among other things,
+a contingency for the case when someone, in an attempt to extend UDT independently, increases the HS Version value to 5
+and tries to test it against SRT. In this case there is nothing to send to the SRT listener, as there is no
+connection established at this point.
 
-Once the Caller gets the SYN cookie from the Listener, it sends the CONCLUSION handshake
-to the Listener.
+If the Encryption Flag field is set to 0 - not advertised, the caller MAY advertise its own cipher and key length.
+If the induction response already advertises a certain value in the Encryption Flag, the caller MAY accept it or force its own value.
+It is RECOMMENDED that if a caller is to be sending the content, then it SHOULD force its own value. If it expects to receive a content from the
+SRT listener, then is it RECOMMENDED that it accepts the value advertised in the Encryption Flag field.
 
-The following values are set by the compliant Caller:
+An alternative behavior MAY be for a caller to take the longer key length in such case.
 
-- Version: 5
-- Handshake Type: CONCLUSION
-- SRT Socket ID: Socket ID of the Caller
-- SYN Cookie: the cookie previously received in the induction phase
-- Encryption Flags: advertised cipher family and block size
-- Extension Flags: a set of flags that define the extensions provided in the handshake
-- The Destination Socket ID in this message is the socket ID that was previously received in the induction phase in the SRT Socket ID field
-of the handshake structure.
+TODO: Receiver TSBPD Delay,  Sender TSBPD Delay.
 
-The Listener responds with the same values shown above, without the cookie (which
-is not needed here), as well as the extensions for HS Version 5 (which will probably be
-exactly the same).
+The SRT Caller forms a Conclusion Request. The following values of a Handshake packet MUST be set by the compliant Caller:
 
-There is not any "negotiation" here. If the values passed in the
-handshake are in any way not acceptable by the other side, the connection will
-be rejected. The only case when the Listener can have precedence over the Caller
+- Destination SRT Socket ID: 0.
+- HS Version: 5.
+- Handshake Type: CONCLUSION.
+- SRT Socket ID: Socket ID of the Caller.
+- SYN Cookie: the listener's cookie from the induction response.
+- Encryption Flags: advertised cipher family and block size.
+- Extension Flags: a set of flags that define the extensions provided in the handshake.
+- The Handshake Extension Message {{handshake-extension-msg}} MUST be present in the conclusion response.
+
+##### The Conclusion Response
+
+The SRT listener receives the Conclusion Request.
+If the values of the conclusion request are in any way not acceptable by the SRT listener side, the connection MUST
+be rejected by sending a Conclusion Response with the Handshake Type field carrying the rejection reason ({{hs-rej-reason}}).
```

Comment:
```suggestion
be rejected by sending a conclusion response with the Handshake Type field carrying the rejection reason ({{hs-rej-reason}}).
```

NOTE: There seems to be some inconsistency in the use of initial caps for such terms as Caller, Listener, Conclusion Response, Cipher Family, Block Size, and others.